scalableminds/webknossos
# Replace usercache by AlfuFoxCache

This is the one remaining usage of the play cache api (the others are replaced in #6903 ).
So this PR contributes to unifying the caching usages.

### URL of deployed dev instance (used for testing):
- https://nomoreplaysynccache.webknossos.xyz

### Steps to test:
- Add logging to cached user request, load some pages, see that there is only one cache miss.

### Issues:
- contributes to #6969
